Src/BootCamp.Chapter/ConsoleLogger.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class ConsoleLogger : ILogger
+    {
+        private string _message;
+
+        public ConsoleLogger()
+        {
+        }
+
+        public ConsoleLogger(string message)
+        {
+            _message = message;
+        }
+
+        public void SetMessage(string message)
+        {
+            _message = message;
+        }
+
+        public string GetMessage()
+        {
+            return _message;
+        }
+
+        public void Log()
```

Comment:
Add one more level to log- LogError

Reply:
I don't fully understand this one. I currently have two targets ConsoleLogger and FileLogger. Should I implement the 3rd as in ErrorLogger or just add a method for each existing targets LogError and display it in different way?

Reply:
I implented LogError in [468c191](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/215/commits/468c19189491938159b027b64832aada3d1513d8)